kolibri/core/tasks/api.py

```diff
@@ -1101,6 +1111,76 @@ def startdeletefacility(self, request):
         resp = _job_to_response(facility_queue.fetch_job(job_id))
         return Response(resp)
 
+    @decorators.action(
+        methods=["post"],
+        detail=False,
+        permission_classes=[
+            IsSuperuser | NotProvisionedCanPost | LODUserHasSyncPermissions
+        ],
+    )
+    def startprovisionsoud(self, request):
```

Comment:
Not required, but this might be a place to try out the new tasks API.

An example is here: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/content/tasks.py#L28

Reply:
Done in https://github.com/learningequality/kolibri/pull/8185/commits/1f5634cf30b3bab33acff3ea1f6f37d67a73f780